jenkins-automation.groovy

```diff
@@ -203,35 +264,36 @@ buildFlowJob('template-base-build-flow') {
 
 // ** DSL Project Builder **
 // ********************
-projectRepos = readFileFromWorkspace("project_repos.txt").split('\n')
-
-job('dsl-project-builder') {
-  using 'template-base'
-  description('This job rebuilds all jobs defined in all jenkins.groovy files whenever any project repository changes.')
-  multiscm {
-    git {
-      remote {
-        url('https://github.com/Ooblioob/jenkins-automation')
-      }
-      branch('*/master')
-    }
-    projectRepos.each { repo ->
-      git {
-        remote {
-          url(repo)
-        }
-        branch('*/master')
-        relativeTargetDir(repo.split('/')[4])
-      }
-    }
-  }
-
-  steps {
-    dsl {
-      external(['**/jenkins.groovy'])
-      additionalClasspath('lib/*.jar')
-      removeAction('DISABLE')
-    }
-  }
-}
-
+// projectRepos = readFileFromWorkspace("project_repos.txt").split('\n')
+// 
+// job('dsl-project-builder') {
+//   using 'template-base'
+//   description('This job rebuilds all jobs defined in all jenkins.groovy files whenever any project repository changes.')
+//   multiscm {
+//     git {
+//       remote {
+//         url('https://github.com/Ooblioob/jenkins-automation')
+//       }
+//       branch('*/master')
+//     }
+//     projectRepos.each { repo ->
+//       git {
+//         remote {
+//           url(repo)
+//         }
+//         branch('*/master')
+//         relativeTargetDir(repo.split('/')[4])
+//       }
+//     }
+//   }
+// 
+//   steps {
+//     dsl {
+//       external(['**/jenkins.groovy'])
+//       additionalClasspath('lib/*.jar')
+//       removeAction('DISABLE')
+//     }
+//   }
+// }
+// 
+// 
```

Comment:
I hesitate to delete only because we don't necessarily know what we'd like to replace this with.  We have plans to replace the project builder / master builder with something that makes more sense and is less cumbersome, but until that is more flushed out I think it makes sense to keep it in there.  If nothing else, it serves as a reminder to tackle that problem and not just ignore it.